Test/Unit/Comment/ApiKeyEndingTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Adyen\Payment\Test\Comment;
+
+use Adyen\Payment\Model\Comment\ApiKeyEnding;
+use Magento\Framework\Encryption\Encryptor;
+
+class ApiKeyEndingTest extends \PHPUnit\Framework\TestCase
+{
+    /**
+     * @var ApiKeyEnding
+     */
+    private $apiKeyEndingComment;
+
+    public function setUp()
+    {
+        $encryptor = $this->createMock(Encryptor::class);
+        $map = [
+            ['4321', '1234'],
+            ['asdfasdfasdf', 'qwerqwerqwer']
+        ];
+        $encryptor->method('decrypt')
+            ->will($this->returnValueMap($map));
+
+        $this->apiKeyEndingComment = new ApiKeyEnding($encryptor);
+    }
+
+    public function testSomething()
```

Comment:
pleeease rename it "toSomethingBetter" ;) so when we read the result we are going to have a clue :D

Reply:
Thank you! I meant to rename that before pushing 😆